lib/MooseX/Types/CyberSource.pm

```diff
@@ -278,6 +290,14 @@ subtype _VarcharSixty, as NonEmptySimpleStr,
   where { length $_ <= 60 },
   message { $varchar_message . '60' };
 
+subtype ShippingMethod, as _VarcharTen,
```

Comment:
I actually thought about this. But SO API specs mentions two separate requirements for shipping method. One is, that max field length is 10, and another, listing possible supported values (none of them reaching max field length).
Enum would only cover supported values requirement. Current implementation covers both.